Makefile.pre.in

```diff
@@ -692,7 +692,10 @@ Modules/Setup: $(srcdir)/Modules/Setup.dist
 		echo "check to make sure you have all the updates you"; \
 		echo "need in your Modules/Setup file."; \
 		echo "Usually, copying Modules/Setup.dist to Modules/Setup will work."; \
+		echo "Or if you want to keep your modifications, touch Modules/Setup"; \
```

Comment:
When Modules/Setup.dist has been changed as for example by 7d895ac953d2ba8c6cb9ade995aef131ddc77ab2 in September 2016, touching Modules/Setup is not sufficient. I suggest to remove that line.